diff --git a/index.js b/index.js
index e797cfe6eb0ef537bfc2deb3ae967a3c680ccbdc..cbd6103e8f564a42229cfe2252a5157c07a2924e 100644
--- a/index.js
+++ b/index.js
@@ -1,9 +1,7 @@
-var Stat = require('fs').Stats
-
 module.exports = cloneStats
 
 function cloneStats(stats) {
-  var replacement = new Stat
+  var replacement = Object.create(Object.getPrototypeOf(stats))
 
   Object.keys(stats).forEach(function(key) {
     replacement[key] = stats[key]
